modules/porous_flow/src/dirackernels/PorousFlowPointEnthalpySourceFromPostprocessor.C

```diff
@@ -0,0 +1,56 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "PorousFlowPointEnthalpySourceFromPostprocessor.h"
+#include "SinglePhaseFluidProperties.h"
+
+registerMooseObject("PorousFlowApp", PorousFlowPointEnthalpySourceFromPostprocessor);
+
+InputParameters
+PorousFlowPointEnthalpySourceFromPostprocessor::validParams()
+{
+  InputParameters params = DiracKernel::validParams();
+  params.addRequiredParam<PostprocessorName>("mass_flux",
+                                             "The postprocessor name holding the mass flux at this "
+                                             "point in kg/s (only positive values are allowed)");
+  params.addRequiredParam<UserObjectName>("fp", "The name of the user object for fluid properties");
```

Comment:
"for fluid properties" -> "used to calculate the fluid properties of the injected fluid"